source/databases/sqlserver/odse/procedures/sp_PublicHealthCaseFact_DATAMART.sql

```diff
@@ -206,6 +206,7 @@ IF OBJECT_ID('NBS_ODSE.DBO.TEMP_PHCPATIENTINFO') IS NOT NULL
 			,RN = ROW_NUMBER() OVER (
 				PARTITION BY PER.PERSON_UID ORDER BY PER.PERSON_UID
 				)
+			, CASE WHEN (CONDITION_CODE.investigation_form_cd NOT LIKE 'PG_%' or CONDITION_CODE.investigation_form_cd  is null) then 1 else 0 end as FLAG1 ---added this as part of optimization (very imp)

```

Comment:
Consider adding a bit more detail on how adding this FLAG will optimize the overall runtime performance.

Reply:
Sure will do in the next commit. May be tomorrow morning ?